File: src/org/jetbrains/agda/core/FunctionBody.java
package org.jetbrains.agda.core;

/**
 * @author Evgeny.Kurbatsky
 */
public class FunctionBody {
    CExpression myLeft;
    CExpression myRight;

    public FunctionBody(CExpression left, CExpression right) {
        this.myLeft = left;
        this.myRight = right;
    }

    @Override
    public String toString() {
        return myLeft + " = " + myRight;
    }
}

File: src/org/jetbrains/agda/core/FunctionBody.kt
package org.jetbrains.agda.core

/**
 * @author Evgeny.Kurbatsky
 */
public open class FunctionBody(val left : CExpression, val right : CExpression) {
    public open fun toString() : String? {
        return left.toString() + " = " + right.toString()
    }

}
